compiler/p/codegen/OMRCodeGenerator.cpp

```diff
@@ -1800,11 +1800,12 @@ bool OMR::Power::CodeGenerator::getSupportsOpCodeForAutoSIMD(TR::CPU *cpu, TR::I
          else
             return false;
       case TR::vdiv:
-      case TR::vneg:
          if (et == TR::Int32 || et == TR::Float || et == TR::Double)
             return true;
-         else
+        else
```

Comment:
Add one more space to this make its indentation consistent the `if` line.
This is my last comment.